0.data_analysis/plate_projection_cp_fs_data/nbconverted/analysis_utils.py

```diff
@@ -0,0 +1,109 @@
+""" This file provides analysis utilities for a variety of tasks """
+
+from sklearn.decomposition import PCA
+import matplotlib.pyplot as plt
+import umap
+import numpy as np
+import pandas as pd
+from scipy.stats import f_oneway
+import itertools
+
+rnd_val = 0  # Random value for all seeds
+rng = np.random.default_rng(seed=rnd_val)  # random number generator
+
+
+def plot_pca(
```

Comment:
I thought I saw these already in #7 ? Do you know what is going on? Maybe the same commits on duplicate PRs?

Reply:
in general I am a big fan of these importable functions!

Reply:
You did, yes, one branch comes from the other. They can be useful, like in these cases!